env.default

```diff
@@ -64,3 +64,8 @@ STRIPE_WEBHOOK_SECRET=
 
 # Sentry
 SENTRY_DSN=
+
+# OIDC
+USE_CONVENTIONAL_AUTH=False
```

Comment:
Feels like we should set this to True here so devs running this and copying env.default to .env just have things work. (Though that should change once django-configurations lands, if we ever stop getting ahead of it)

Reply:
```suggestion
USE_CONVENTIONAL_AUTH=True
```
Good point